ch_backup/clickhouse/control.py

```diff
@@ -182,7 +182,8 @@
         files Array(String),
         tarball Bool,
         disk_name String,
-        verified Bool
+        verified Bool,
+        encrypted Nullable(Bool)
```

Comment:
Why `Nullable` and it seems [here](https://github.com/yandex/ch-backup/blob/849cd5b9135b0e5915dd7e21331a469b81d9a900/ch_backup/clickhouse/control.py#L1009) we should do DROP instead of TRUNCATE to be able to change the schema of this temporal table.

Reply:
Nullable for backward compatibility (because previous tables do not contain this field)

Reply:
Actually `_deduplication_info` table is temporal and recreated every run. I propose not to make this column just `Bool`